model/host/db.go

```diff
@@ -913,8 +916,8 @@ func Aggregate(ctx context.Context, pipeline []bson.M, options ...*options.Aggre
 }
 
 // Count returns the number of hosts that satisfy the given query.
-func Count(ctx context.Context, query bson.M) (int, error) {
-	res, err := evergreen.GetEnvironment().DB().Collection(Collection).CountDocuments(ctx, query)
+func Count(ctx context.Context, query bson.M, opts *options.CountOptions) (int, error) {
```

Comment:
You can avoid having to pass `nil` to every `Count` call that uses no special options by making it a variadic parameter:
```go
func Count(ctx context.Context, query bson.M, opts ...*options.CountOption) {
    res, err := evergreen.GetEnvironment().DB().Collection(Collection).CountDocuments(ctx, query, opts...)
```

Reply:
Thanks!